utils/grid_visualizer/README.md

```diff
@@ -0,0 +1,3 @@
+# Grid visualizer
+
+This is a python utility which can visualize tile grid. It can load the grid definition from various sources (prjxray database files, arch.xml, rr_graph.xml, SQL database) and render it to a SVG/PDF file.
```

Comment:
Can you add the following to this README file?

 - [ ] A little screen showing the output
 - [ ] Some sample command lines which can be run

Reply:
I updated the README file. How should I add screens ? Simply as files ?

Reply:
Yeah, just a couple of png or jpg files.